### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by okoyecharles**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  </head>
+
+  <body>
+    <header id="header">
+      <img src="https://www.google.com/search?q=image+of+headphone&rlz=1CAZAQH_enNG975&sxsrf=APq-WBuwS8F0KaP4AmHrYaDb2sSPQnEl8w:1648298707986&tbm=isch&source=iu&ictx=1&vet=1&fir=mIT4NCgfDXtE1M%252Ch5tqX_QHG5wDXM%252C_%253BgsCGxq3lpLsMWM%252Ch5tqX_QHG5wDXM%252C_%253B5BNEhzqLR8Dc3M%252Ch5tqX_QHG5wDXM%252C_%253BSqta8j3U9KnYJM%252C-wyP6cFsRWXx7M%252C_%253BVGj7RF0Q6tYZ6M%252C-wyP6cFsRWXx7M%252C_%253B7Hn9sHMC2jC5-M%252CG1miClKgJgnM-M%252C_%253B5L7yigwMDOgX0M%252Ch5tqX_QHG5wDXM%252C_%253BcHHelTGnHkyADM%252CG1miClKgJgnM-M%252C_%253B0H7OSFnnaPMuIM%252CG1miClKgJgnM-M%252C_%253Bwb1KJQBRuykx4M%252C-wyP6cFsRWXx7M%252C_%253BhHVNOMYfMCE-cM%252Ch5tqX_QHG5wDXM%252C_%253B3-xY1ayV8GzawM%252CuvtEsPaVDI4FUM%252C_%253BmmIGoVH3_psyrM%252CeMa4xi-J2781NM%252C_%253BnMt1CHAO49p7FM%252CY7LLjHEJRvCexM%252C_&usg=AI4_-kTzMrwndBrh_DT1ycL_xHAt6KEtTw&sa=X&ved=2ahUKEwi76-OX5-P2AhUihf0HHeHsCX8Q9QF6BAgpEAE#imgrc=gsCGxq3lpLsMWM" id="header-img">
+      <nav id="nav-bar">
+          <a href="#header" class="nav-link">About</a>
+          <a href="#header" class="nav-link">Contact us</a>
+          <a href="#header" class="nav-link">More products</a>
```
Comment (okoyecharles) [inline index.html:15]: please try using an unordered list for your navigation links. It helps in the organization and identification of the links

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      </nav>
+    </header>
+    <video src="https://www.youtube.com/watch?v=sBba4MGPc2U" id="video"></video>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input name="email" type="email" id="email" placeholder="Input your email here">
+      <button id="submit"><a href="https://www.freecodecamp.com/email-submit"> </a></button>
```
Comment (okoyecharles) [inline index.html:21]: Please try defining an input with a ' type="button" ' attribute. It helps browsers in identifying the functionality of the tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (okoyecharles) [inline index.html:37]: i recommend you delete the HTML comment as requested by the company

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (okoyecharles) [inline index.html:5]: Please try changing this to a more descriptive title

**Review by okoyecharles — COMMENTED** [review]
# Hi @OkoyeCharles  ✋, 

While you made a great effort in this project, there are some changes required of you, check the comments in the PR thread

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏
--------

**Review by Kandy-Peter — APPROVED** [review]
Hi @OkoyeCharles ,

# APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏

Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**